test/backend_tests/test_utils.py

```diff
@@ -0,0 +1,46 @@
+"""Tests
+
+How to run:
+    python -m unittest discover
+"""
+import os
+import sys
+import unittest
+from pathlib import Path
+
+TEST_DIR = os.path.dirname(__file__)
+PROJECT_ROOT = Path(TEST_DIR).parent.parent
+sys.path.insert(0, str(PROJECT_ROOT))
+from backend.db.utils import get_db_connection, insert_fetch_status, run_sql, sql_query
+
+MOCK_FAILED_FETCH_OBJS = []
+for codeset_id in [1, 2, 3]:
+    MOCK_FAILED_FETCH_OBJS.append({'table': 'code_sets', 'primary_key': codeset_id, 'status_initially':
+        'fail-excessive-items', 'comment': 'Unit testing.'})
+    MOCK_FAILED_FETCH_OBJS.append({'table': 'code_sets', 'primary_key': codeset_id, 'status_initially':
+        'fail-excessive-members', 'comment': 'Unit testing.'})
+    MOCK_FAILED_FETCH_OBJS.append({'table': 'code_sets', 'primary_key': codeset_id, 'status_initially':
+        'fail-0-members', 'comment': 'Unit testing.'})
+
+
+class TestBackendUtils(unittest.TestCase):
```

Comment:
Created new test file for testing `backend/db/utils.py` and added some tests for the new `fetch_audit` table.

Reply:
@hugwuoke Can you take a look at this? I just want you to be aware with the style I went with for testing this new code that I was working on. I learned a little in the process, kind of a new pattern.

First, I did what we're starting to do with unit tests now where we have a different `test_` file dedicated to its corresponding file in the source code. Then, I made the class name... which is kinda verbose... Idk if this is the best way to do it: test class `TestBackendDbUtilsFetchAudit`. The reason I put `FetchAudit` at the end is just for further grouping. These are related functions. And also they have a common setup/teardown step, where before any tests in the class run, it inserts some mock data into the DB. Then, at the end, it removes that mock data.

This PR will probably get merged before your test PR, so we can `rebase` your test PR branch so you can get these new tests.